### ISSUE TITLE
Error missing node even though installed

### ISSUE THREAD

**Issue by hainguyen666** [cdb-boop/ComfyUI-Bringing-Old-Photos-Back-to-Life#18]
![image](https://github.com/user-attachments/assets/89710816-26bc-40df-960b-08ef27388ee9)
![image](https://github.com/user-attachments/assets/006500f5-a6d7-4667-aad9-475793aeb1be)
Hello everyone, can anyone tell me what my problem is? I have fully installed it but it still says node is missing, please help, thank you.

**Comment by cdb-boop** [comment]
What was the error printed in the command line when the node failed to import?

**Comment by Jmagick2033** [comment]
It is probably the same as mine.` note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for dlib
Failed to build dlib
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (dlib)`

**Comment by cdb-boop** [comment]
dlib is used to detect faces to crop out and restore in a separate pass.

If dlib is the issue, you can try to follow what I wrote on [troubleshooting](https://github.com/cdb-boop/ComfyUI-Bringing-Old-Photos-Back-to-Life?tab=readme-ov-file#installation-troubleshooting). You can either try install/build with pip manually to fix it, or you can use a prebuilt wheel online, for example, [here](https://github.com/sachadee/Dlib) or [here](https://github.com/z-mahmud22/Dlib_Windows_Python3.x) (I have no idea how secure it is to use).

**Comment by hainguyen666** [comment]
> What was the error printed in the command line when the node failed to import?

![image](https://github.com/user-attachments/assets/756f549c-83f9-457e-a90e-d4baaa0c19a0)

**Comment by cdb-boop** [comment]
I need to see the error before this.

**Comment by hainguyen666** [comment]
> > What was the error printed in the command line when the node failed to import?
> 
> ![image](https://private-user-images.githubusercontent.com/143708202/361186021-756f549c-83f9-457e-a90e-d4baaa0c19a0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ1NzA0NTMsIm5iZiI6MTcyNDU3MDE1MywicGF0aCI6Ii8xNDM3MDgyMDIvMzYxMTg2MDIxLTc1NmY1NDljLTgzZjktNDU3ZS1hOTBlLWQ0YmFhYTBjMTlhMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgyNVQwNzE1NTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mZWUzMDQ3OGI3ZjEwZjAxMmQxMzFkZGU0NzJjZTVlMTk2YmEwMTIxODZiMzk2ODY3ODAwY2M1ZDlkNzM1MTVhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.1e8rnBZHdI5i-qLGHwPH6gIcYUXmvesb8mRhuyPNC2s)


![image](https://github.com/user-attachments/assets/5a63b4ec-815f-40b7-9152-82d76f529a27)

**Comment by cdb-boop** [comment]
Yes, the problem is a dlib issue. Again, you'll need to follow the troubleshooting instructions.

```shell
D:\ComfyUI_windows_portable\python_embeded\python.exe -m pip install easydict matplotlib opencv-python scikit-image scipy

D:\ComfyUI_windows_portable\python_embeded\python.exe -m pip install cmake
D:\ComfyUI_windows_portable\python_embeded\python.exe -m pip install dlib
```

If this doesn't work, then maybe you can try a prebuilt dlib like the ones found [here](https://github.com/z-mahmud22/Dlib_Windows_Python3.x) or [here](https://github.com/sachadee/Dlib), and then install with something like the following:

```shell
D:\ComfyUI_windows_portable\python_embeded\python.exe -m pip install path\to\dlib.whl
```
